docs/examples/using-nonstandard-fresh.md

```diff
@@ -0,0 +1,78 @@
+---
+description: |
+  For cases where the latest release doesn't fit your needs.
+---
+
+Pretend you have a use case where you need to modify your project to use a
+nonstandard version of Fresh. Or you want to use a slightly different
+initialization script. This page has you covered.
+
+## Nonstandard Fresh in `deno.json`
+
+### Latest commit
+
+Let's say you like living life in the fast lane, and want the latest and
+greatest from `main`. How can you modify your project to no longer use the
+current release, but instead the most recent commit? Just make the following
+changes to your `deno.json`:
+
+```diff
+     "update": "deno run -A -r https://fresh.deno.dev/update ."
+   },
+   "imports": {
+-    "$fresh/": "https://deno.land/x/fresh@1.2.0/",
++    "$fresh/": "https://raw.githubusercontent.com/denoland/fresh/main/",
+     "preact": "https://esm.sh/preact@10.15.1",
+     "preact/": "https://esm.sh/preact@10.15.1/",
+     "preact-render-to-string": "https://esm.sh/*preact-render-to-string@6.1.0",
+```
+
+### Forked Fresh
+
+Or what if you have a PR created but it's not getting merged into `main`. Don't
+worry, you can use the same approach to reference any branch in a fork as well.
+Here's an example of referencing a feature in a forked repository that hasn't
+been merged yet (at the time of writing this):
+
+```diff
+     "update": "deno run -A -r https://fresh.deno.dev/update ."
+   },
+   "imports": {
+-    "$fresh/": "https://deno.land/x/fresh@1.2.0/",
++    "$fresh/": "https://raw.githubusercontent.com/deer/fresh/state_in_props/",
+     "preact": "https://esm.sh/preact@10.15.1",
+     "preact/": "https://esm.sh/preact@10.15.1/",
+     "preact-render-to-string": "https://esm.sh/*preact-render-to-string@6.1.0",
+```
+
+## Creating a new project
+
+What if you're getting into open source development, and you've of course
+decided to contribute to the best, freshest project around. Maybe you want to
+create a test project based on your local changes.
+
+### Creating a project from source
+
+Instead of doing it like this:
+
+```sh
+deno run -A -r https://fresh.deno.dev/
+```
+
+do it like this:
+
+```sh
+deno run -A -r ~/code/fresh/init.ts
```

Comment:
How do you feel about changing this to `path/to/fresh/init.ts`? Just as an additional hint that this is an example path.

Reply:
sounds good